services/sensiolabs/sensiolabs.tester.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+// const Joi = require('joi')
```

Comment:
Planning to use this (plus the regexes on lines 9 and 18) as part of the live tests which still need to be written